ups/fife_utils.table

```diff
@@ -35,7 +35,7 @@ Action=setup
     ExeActionRequired(setup_ifdhc_if_not_setup)
     ExeActionRequired(setup_sam_web_client_if_not_setup)
     SetupOptional(jobsub_client)
-    SetupOptional(poms_client)
+    #SetupOptional(poms_client)
```

Comment:
Is this intentional?

Reply:
Yes, I should have mentioned this.   Long ago , when Nova was submitting jobs outside of POMS but wanting to monitor them in POMS, we added a config section to fife_launch to suppoirt this, and a dependency on poms_client. 
So this has since been dropped, but we still had the dependency, which is causing the UPS side to pick up dependencies like cigetcert, which hasn't been updated in UPS for quite a while, and now gets you an old sl7 executable that doesn't work...   That should probably have been in a different pull request, but I had this checked out when I was doing it...